hikari/guilds.py

```diff
@@ -407,6 +407,21 @@ def presence(self) -> typing.Optional[presences_.MemberPresence]:
         """
         return self.app.cache.get_presence(self.guild_id, self.id)
 
+    @property
+    def roles(self) -> typing.Sequence[Role]:
+        """Return the roles the user has.
+
+        This will be empty if the roles are missing from the cache.
+
+        Returns
+        -------
+        typing.Sequence[hikari.guilds.Role]
+            The roles the users has.
+        """
+        roles_view = self.app.cache.get_roles_view_for_guild(self.guild_id)
+
+        return [r for r in roles_view.values() if r.id in self.role_ids]
```

Comment:
test coverage for this?

Reply:
Will move the test coverage, i just forgot to do it